EnglishTranslation-master/QUEST.tsv

```diff
@@ -5047,7 +5047,7 @@ QUEST_20161214_005042	There is nothing written on the board.	경고문에는 아
 QUEST_20161214_005043	Goddess Zemyna is always looking after us.{nl}My brother also says She is always protecting us.	제미나 여신께선 언제나 저흴 지켜주고 계세요.{nl}제 오빠도 제미나 여신께서 돌봐 주시고 있을 거라 믿고 있어요.				소녀
 QUEST_20161214_005044	{nl}Oh, you went to see the Statue of Goddess Gabija?{nl}Is She listening to our prayers?	{nl}아, 여행자님은 가비야 여신을 만나고 오셨다구요?{nl}가비야 여신께서는 기도를 들어주실까요?				소녀
 QUEST_20161214_005045	The way I see it, we need to find the truth about our history no matter what.{nl}But I can't help thinking: what if the forgotten truths are more than the ones we know?	역사의 진실은 뒤늦게라도 꼭 밝혀져야 한다고 봐.{nl}하지만 밝혀진 진실보다는, 그냥 묻히고 잊혀진 진실이 훨씬 많지 않을까?				파이로맨서 마스터
-QUEST_20161214_005046	{nl}That's why I'm so glad Ruklys'?壤밸??쳋e is finally clean.{nl}I assume you read the book I wrote?	{nl}그런 의미에서 루클리스의 누명이 벗겨진 건 참 다행이야.{nl}내가 적어 둔 책은 읽어 봤겠지?				파이로맨서 마스터
+QUEST_20161214_005046	{nl}That's why I'm so glad Ruklys' offense is finally clean.{nl}I assume you read the book I wrote?{nl}	그런 의미에서 루클리스의 누명이 벗겨진 건 참 다행이야.{nl}내가 적어 둔 책은 읽어 봤겠지?				파이로맨서 마스터

```

Comment:
`That's why I'm so glad Ruklys' offense is finally clean` -> `That's why I'm so glad Ruklys' name is finally cleared.`